src/client/metric.rs

```diff
@@ -107,6 +99,35 @@ pub enum Space {
     EByte
 }
 
+impl Space {
+    fn from_u8(x: u8) -> Option<Self> {
+        match x {
+            0 => Some(Space::Byte),
```

Comment:
@ryandoyle It would be better to force the use of enums over constants, even internally, since Rust mandates we handle all of its variants, and like @suyash said it also gives us the proper string representation.